packages/db/prisma/schema.prisma

```diff
@@ -6,7 +6,7 @@ generator zod {
   provider = "zod-prisma-types"
   output   = "../src/schemas"
 
-  useMultipleFiles = true
+  useMultipleFiles = false
```

Comment:
Node does not support directory import, and this codegenerator generates directory import when using multiple files.